internal/plan/plan.go

```diff
@@ -105,6 +149,15 @@ const (
 	ReplaceOverride ServiceOverride = "replace"
 )
 
+type ServiceAction string
+
+const (
+	ActionUnset      ServiceAction = ""
+	ActionRestart    ServiceAction = "restart"
+	ActionExitPebble ServiceAction = "exit-pebble"
```

Comment:
Per earlier note, let's please review this before it's final.

Reply:
Per discussion, updated to just "exit".